301-multi-tier-loadbalancing/azuredeploy.json

```diff
@@ -0,0 +1,1073 @@
+      "minLength": 1,
+      "metadata": {
+        "description": "Traffice Manager profile name"
+      }
+    },
+    "adminuser": {
+      "defaultValue": "admin123",
+      "type": "string",
+      "minLength": 1,
+      "metadata": {
+        "description": "Admin username"
+      }
+    },
+    "adminuserPassword": {
+      "defaultValue": "TestUser@123",
+      "type": "securestring",
+      "minLength": 1,
+      "metadata": {
+        "description": "Admin Password"
+      }
+    },
+    "App-prefix": {
+      "defaultValue": "ab",
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 3,
+      "metadata": {
+        "description": "Application prefix name, should be < 3 characters"
+      }
+    },
+
+    "imageSKU": {
+      "type": "string",
+      "defaultValue": "2012-R2-Datacenter",
+      "allowedValues": [
+        "2008-R2-SP1",
+        "2012-Datacenter",
+        "2012-R2-Datacenter"
+      ],
+      "metadata": {
+        "description": "The Windows version for the VM"
+      }
+    },
+    "vmSize": {
+      "type": "string",
+      "minLength": 1,
+      "defaultValue": "Standard_D1_v2",
+      "allowedValues": [
+        "Basic_A0",
+        "Basic_A1",
+        "Basic_A2",
+        "Basic_A3",
+        "Basic_A4",
+        "Standard_A0",
+        "Standard_A1",
+        "Standard_A2",
+        "Standard_A3",
+        "Standard_A4",
+        "Standard_A5",
+        "Standard_A6",
+        "Standard_A7",
+        "Standard_D1_v2",
+        "Standard_D2_v2",
+        "Standard_D3_v2",
+        "Standard_D4_v2",
+        "Standard_D11_v2",
+        "Standard_D12_v2",
+        "Standard_D13_v2",
+        "Standard_D14_v2"
+      ],
+      "metadata": {
+        "description": "Size of the virtual machine, must be available in the virtual machine's location"
+      }
+    },
+    "numberOfVMInstances": {
+      "type": "int",
+      "minValue": 1,
+      "maxValue": 10,
+      "defaultValue": 2,
+      "metadata": {
+        "description": "Number of VM instances to be created behind internal load balancer control"
+      }
+    },
+    "loadbalancer-dns-prefix": {
+      "type": "string",
+      "minLength": 1,
+      "maxLength": 10,
+      "defaultValue": "lb",
+      "metadata": {
+        "description": "loadbalancer dns name should be lowercase letters!"
+      }
+    },
+    "appGW-skuName": {
+      "type": "string",
+      "allowedValues": [
+        "Standard_Small",
+        "Standard_Medium",
+        "Standard_Large"
+      ],
+      "defaultValue": "Standard_Small",
+      "metadata": {
+        "description": "Sku Name"
+      }
+    },
+    "appGW-capacity": {
+      "type": "int",
+      "minValue": 1,
+      "maxValue": 10,
+      "defaultValue": 4,
+      "metadata": {
+        "description": "Number of Application Gateway instances"
+      }
+    },
+    "appGW-pathMatch1": {
+      "type": "string",
+      "metadata": {
+        "description": "Path match string for Path Rule 1"
```

Comment:
Is this description descriptive enough that people will know what to specify for it? It doesn't have a default; I don't know what I would put here...